components/meter/demo/meter.html

```diff
@@ -22,6 +33,10 @@ <h2>Meter Linear</h2>
 			<d2l-demo-snippet>
 				<d2l-meter-linear value="2" max="15" style="width: 250px" text="Activities" text-inline></d2l-meter-linear>
 				<d2l-meter-linear value="3" max="6" style="width: 200px" text="Visited: {x/y}" percent></d2l-meter-linear>
+				<div class="dark-background-1">
```

Comment:
Nit: since you're not actually using these classes multiple times, you could just do inline CSS `<div style="background-color: darkblue;">`